packages-common/pluggable-widgets-tools/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
-    "extends": "../configs/tsconfig.base",
+    "extends": "./configs/tsconfig.base",
     "baseUrl": "./",
-    "include": ["./"],
+    "include": ["./src"],
     "compilerOptions": {
         "target": "es2015",
         "lib": ["es2015"],
```

Comment:
I think most of the settings in this files either duplicate one from base config or a default value, so they can be removed.

Reply:
Cleaned up tsconfig.